testdata/baselines/reference/submoduleAccepted/conformance/checkExportsObjectAssignProperty.errors.txt.diff

```diff
@@ -0,0 +1,147 @@
+     m1.readonlyProp = "name";
+-       ~~~~~~~~~~~~
+-!!! error TS2540: Cannot assign to 'readonlyProp' because it is a read-only property.
+     m1.readonlyAccessor = 12;
+-       ~~~~~~~~~~~~~~~~
+-!!! error TS2540: Cannot assign to 'readonlyAccessor' because it is a read-only property.
+     m1.thing = "no";
+-    ~~~~~~~~
+-!!! error TS2322: Type 'string' is not assignable to type 'number'.
+     m1.rwAccessors = "no";
+-    ~~~~~~~~~~~~~~
+-!!! error TS2322: Type 'string' is not assignable to type 'number'.
+     m1.setonlyAccessor = 0;
+-    ~~~~~~~~~~~~~~~~~~
+-!!! error TS2322: Type 'number' is not assignable to type 'string'.
+     
+     import m2 = require("./mod2");
++                        ~~~~~~~~
++!!! error TS2306: File 'mod2.js' is not a module.
+     
+     m2.thing;
+     m2.readonlyProp;
+@@= skipped -30, +22 lines =@@
+     
+     // disallowed assignments
+     m2.readonlyProp = "name";
+-       ~~~~~~~~~~~~
+-!!! error TS2540: Cannot assign to 'readonlyProp' because it is a read-only property.
+     m2.readonlyAccessor = 12;
+-       ~~~~~~~~~~~~~~~~
+-!!! error TS2540: Cannot assign to 'readonlyAccessor' because it is a read-only property.
+     m2.thing = 0;
+-    ~~~~~~~~
+-!!! error TS2322: Type 'number' is not assignable to type 'string'.
+     m2.rwAccessors = "no";
+-    ~~~~~~~~~~~~~~
+-!!! error TS2322: Type 'string' is not assignable to type 'number'.
+     m2.setonlyAccessor = 0;
+-    ~~~~~~~~~~~~~~~~~~
+-!!! error TS2322: Type 'number' is not assignable to type 'string'.
+     
+-==== mod1.js (0 errors) ====
++==== mod1.js (5 errors) ====
+     Object.defineProperty(exports, "thing", { value: 42, writable: true });
++                          ~~~~~~~
++!!! error TS2304: Cannot find name 'exports'.
+     Object.defineProperty(exports, "readonlyProp", { value: "Smith", writable: false });
++                          ~~~~~~~
++!!! error TS2304: Cannot find name 'exports'.
+     Object.defineProperty(exports, "rwAccessors", { get() { return 98122 }, set(_) { /*ignore*/ } });
++                          ~~~~~~~
++!!! error TS2304: Cannot find name 'exports'.
+     Object.defineProperty(exports, "readonlyAccessor", { get() { return 21.75 } });
++                          ~~~~~~~
++!!! error TS2304: Cannot find name 'exports'.
+     Object.defineProperty(exports, "setonlyAccessor", {
++                          ~~~~~~~
++!!! error TS2304: Cannot find name 'exports'.
+         /** @param {string} str */
+         set(str) {
+             this.rwAccessors = Number(str) 
+         }
+     });
+     
+-==== mod2.js (0 errors) ====
++==== mod2.js (5 errors) ====
+     Object.defineProperty(module.exports, "thing", { value: "yes", writable: true });
++                          ~~~~~~
++!!! error TS2580: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
+     Object.defineProperty(module.exports, "readonlyProp", { value: "Smith", writable: false });
```

Comment:
Seemingly this is broken because the new code does not yet handle `defineProperty` based exports. But, it's not clear to me why this succeeded in the past.